src/org/traccar/reports/Summary.java

```diff
@@ -60,7 +63,14 @@ private static SummaryReport calculateSummaryResult(long deviceId, Date from, Da
                 speedSum += position.getSpeed();
                 result.setMaxSpeed(position.getSpeed());
             }
-            result.setDistance(ReportUtils.calculateDistance(firstPosition, previousPosition));
+            boolean ignoreOdometer = false;
+            if (device.getAttributes().containsKey(ReportUtils.IGNORE_ODOMETER)) {
```

Comment:
There is function lookupAttribute https://github.com/tananaev/traccar/blob/master/src/org/traccar/database/DeviceManager.java#L319
But it looks through device->group->server.
Two options:
1. Use it and may be do not use config because there is server attributes.
2. Add additional parameters to function to ignore groups/server

What is better? I like first one. It will allow to configure whole group (if enabled)